src/Microsoft.AspNet.Mvc.Core/ModelBinding/FormFileModelBinder.cs

```diff
@@ -48,8 +48,10 @@ public class FormFileModelBinder : IModelBinder
                         SuppressValidation = true,
                     };
 
-                var valueProviderResult = new ValueProviderResult(rawValue: value);
-                bindingContext.ModelState.SetModelValue(bindingContext.ModelName, valueProviderResult);
+                bindingContext.ModelState.SetModelValue(
+                    bindingContext.ModelName,
+                    rawValue: null,
+                    attemptedValue: null);
```

Comment:
Decided to split this and the body cases. No action needed here.

See https://github.com/aspnet/Mvc/pull/3011#discussion-diff-37593962 for more info.